cloudfoundry-client/src/main/java/org/cloudfoundry/client/v2/serviceinstances/ServiceInstances.java

```diff
@@ -24,12 +24,19 @@
 public interface ServiceInstances {
 
     /**
-     * Makes the <a href="http://apidocs.cloudfoundry.org/214/service_instances/list_all_service_instances.html">List
-     * Service Instances</a> request
+     * Makes the <a href="http://apidocs.cloudfoundry.org/214/service_instances/list_all_service_instances.html">List Service Instances</a> request
      *
      * @param request the List Service Instances request
      * @return the response from the List Service Instances request
      */
     Publisher<ListServiceInstancesResponse> list(ListServiceInstancesRequest request);
 
+
+    /**
+     * Makes the <a href="http://apidocs.cloudfoundry.org/214/service_instances/retrieve_a_particular_service_instance.html">Retrieve a Particular Service Instance</a> request
+     *
+     * @param request the Get Service Instance request
+     * @return the response from the Get Service Instance request
+     */
+    Publisher<ServiceInstanceResource> get(GetServiceInstanceRequest request);
```

Comment:
This should return a `GetServiceInstanceResponse`.  That type can be a straight extension of `ServiceInstanceResource` (not adding anything), but we want the abstraction in place to give us flexibility.